src/OpenTelemetry.Instrumentation.Hangfire/Implementation/HangfireInstrumentationJobFilterAttribute.cs

```diff
@@ -32,13 +32,13 @@ internal sealed class HangfireInstrumentationJobFilterAttribute : JobFilterAttri
 {
     private readonly HangfireInstrumentationOptions options;
 
+#pragma warning disable CA1019 // Define accessors for attribute arguments
```

Comment:
Is this just about making `options` a property instead of a field? I think we could simply fix that if that's the case.

Reply:
This construction `private HangfireInstrumentationOptions Options { get; }` leads to: `CA1019	If Options is the property accessor for positional argument options, make it public	OpenTelemetry.Instrumentation.Hangfire`.

Which is nob better. I do not think that we should make it public.